unit_tests/conftest.py

```diff
@@ -77,8 +77,16 @@ def fixture_docker_scylla(request: pytest.FixtureRequest):
             os.chdir(curr_dir)
 
     ssl_dir = (Path(__file__).parent.parent / 'data_dir' / 'ssl_conf').absolute()
+    # experimental features previous enabled by '--experimental' umbrella flag, which
+    # was deprecated and removed, so let's specify them explictly. 'udf' should be
+    # enabled separately along with 'enable_user_defined_functions'
+    experimental_features = ['alternator-streams',
+                             'alternator-ttl',
+                             'keyspace-storage-options']
+    experimental_flags = ' '.join(f'--experimental-features {feature}'
+                                  for feature in experimental_features)
     scylla = RemoteDocker(LocalNode("scylla", cluster), image_name=docker_version,
```

Comment:
let's not touch this part in less we have too, we don't need the experimental flags here at all

Reply:
@nyh BTW we might want to remove `--experimental` from scylla docker as well...

Reply:
actually, i think the foremost important thing is not to do the cleanup. but to fix the test failure -- that's why i jumped in and prioritize it.